spec/factories/access_control_lists.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryGirl.define do
   factory :access_control_list do
-    user_group
+    association :user_group, factory: :user_group_with_users
```

Comment:
I don't think this needs to be the default factory? Best to just build the one you need when you need it. 

For ref, if this is the default, everywhere the `access_control_list` factory is used it'll go an build 2 extra users, identity groups & memberships. This chained factory building just adds time to the spec runs and most likely isn't needed for the rest of the specs.